matrix_sdk_common/Cargo.toml

```diff
@@ -12,10 +12,9 @@ version = "0.1.0"
 
 [dependencies]
 js_int = "0.1.5"
-ruma-api = "0.16.1"
-ruma-client-api = "0.9.0"
-ruma-events = "0.21.2"
-ruma-identifiers = "0.16.1"
+
+ruma = { git = "https://github.com/ruma/ruma", features = ["client-api", "rand"] }
```

Comment:
Do we need the rand feature?

Reply:
We don't but, rumatui does use it. I can turn it on by depending on the same crate with the feature enabled in rumatui.

Reply:
Out of interest what are you using the rand feature for?

Reply:
To deal with local message echo I use a "fake" `EventId` for my internal `Message` stuct which is sent to the UI to be displayed and reconstructed for the send_message request. Although this could be accomplised by `EventId::try_from("$rumatui:fake_event.com").unwrap()`. We also use it in our tests for some reason, I could also change this.

Reply:
I would think that using the transaction ID for this would be the safer thing to do. That way if fail to get the event id from the message sending request you can get it from the sync, e.g. if the request fails on your side but succeeds on the server side.

Reply:
Oh yea I do use the transaction Id to actually filter and compare the EventId it is just to fill out the struct, it would be painful in a few places to make it an `Option` but I certainly could.